desktop/packages/linux/rs.ruffle.Ruffle.desktop

```diff
@@ -23,6 +23,7 @@ GenericName[ja]=Flashプレイヤー
 GenericName[ko]=플래시 플레이어
 GenericName[nl]=Flash speler
 GenericName[pl]=Odtwarzacz Flash
+GenericName[pt_PT]=Flash Player
```

Comment:
Not sure about this translation: "Player" is translatable here, it should be consistent with the keywords (where "reprodutor" is used).

Reply:
So it's capitalized in the source because ... title-case? 🤔

Reply:
yes

Reply:
I see - made a comment about it on Crowdin, and undone this translation.